spec/models/seat_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Seat, type: :model do
+  context 'when creating a Seat' do
+    let(:seat) {FactoryBot.build(:seat)}
+    let(:bus) {FactoryBot.build(:bus)}
+
+    it 'ensure seat number presence' do
+      seat.number = nil
```

Comment:
I guess using the following format will be easier. 
`it { is_expected.to validate_presence_of(:number) }`

Reply:
now using shoulda-matcher for tests.